docs/playbooks/oneclick-core.md

```diff
@@ -48,9 +48,14 @@ The other values can be left at their defaults.  Finally, click **Generate** and
 - `MAS_CONFIG_DIR` Directory where generated config files will be saved (you may also provide pre-generated config files here)
 - `SLS_LICENSE_ID` The license ID must match the license file available in `SLS_LICENSE_FILE`
 - `SLS_LICENSE_FILE` The path to the location of the license file.
-- `UDS_CONTACT_EMAIL` Defines the email for person to contact for UDS
-- `UDS_CONTACT_FIRSTNAME` Defines the first name of the person to contact for UDS
-- `UDS_CONTACT_LASTNAME` Defines the last name of the person to contact for UDS
+- `BAS_ACTION` Defines UDS or DRO operator to install. Default: UDS [Values: DRO, UDS]
```

Comment:
Not sure this is an "action" but more of a "provider" or "type". We don't want to get this confussed with the install, uninstall actions.